src/load-saints.ts

```diff
@@ -1,4 +1,4 @@
-import { LiturgicalClass } from "./types";
+import { LiturgicalClass } from "../src/types"; //NOTE intentional, must resolve from dist/
```

Comment:
This is a confusing solution.
It would be best to copy the types file to the dist folder

Reply:
I'm pretty sure we'd have to eat the tradeoff that we saw in #1 if we want to hard-copy the file, where:
> we are forced to add a dev dependency (`cpx`) [and additional build step] for copying the files over. Writing a simple `cp` or `rsync` would work on POSIX systems, but not Windows.

Are you ok with taking that tradeoff? Or is there another option here that you think I should try/think about first?

Reply:
Yes, I guess that tradeoff is fine.